waku/v2/protocol/rln/contracts/import.json

```diff
@@ -2,27 +2,27 @@
   "language": "Solidity",
   "sources": {
     "WakuRln.sol": {
-      "urls": ["./waku-rln-contract/contracts/WakuRln.sol"]
+      "urls": ["../../../../../libs/waku-rln-contract/contracts/WakuRln.sol"]
```

Comment:
Would be nice if we could use some env variable instead of all these ..'s

Reply:
Tracking this request in https://github.com/waku-org/go-waku/issues/789